src/status_im/ui/screens/wallet/views.cljs

```diff
@@ -88,8 +88,9 @@
                           [react/refresh-control {:on-refresh #(re-frame/dispatch [:update-wallet])
                                                   :tint-color :white
                                                   :refreshing false}])}
-      [react/view {:style styles/scroll-top}] ;; Hack to allow different colors for top / bottom scroll view]
       [total-section portfolio-value]
       [list/action-list actions
        {:container-style styles/action-section}]
-      [asset-section assets]]]))
+      [asset-section assets]
+      [react/view {:style styles/scroll-bottom}] ;; Hack to allow different colors for top / bottom scroll view]
+      ]]))
```

Comment:
https://github.com/bbatsov/clojure-style-guide#gather-trailing-parens
@psdp, please place all trailing parentheses on a single line instead of distinct lines.